README.md

```diff
@@ -43,6 +43,16 @@ You can create an SAP Integration Suite, advanced event mesh service with the fo
 }
 ```
 
+Additional configuration options are:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `session` | [SessionProperties](https://docs.solace.com/API-Developer-Online-Ref-Documentation/nodejs/solace.SessionProperties.html) | Used for [`createSession`](https://docs.solace.com/API-Developer-Online-Ref-Documentation/nodejs/solace.SolclientFactory.html#createSession) |
+| `queue`   | [createMsgVpnQueue](https://docs.solace.com/API-Developer-Online-Ref-Documentation/swagger-ui/software-broker/config/index.html#/msgVpn/createMsgVpnQueue) | The queue object which is created via the SEMP API |
+| `consumer` | [MessageConsumerProperties](https://docs.solace.com/API-Developer-Online-Ref-Documentation/nodejs/solace.MessageConsumerProperties.html) | Used for [`createMessageConsumer`](https://docs.solace.com/API-Developer-Online-Ref-Documentation/nodejs/solace.Session.html#createMessageConsumer) |
+
+The default values can be found [in the package.json](https://github.com/cap-js/advanced-event-mesh/blob/main/package.json#L35).
```

Comment:
```suggestion
The default values can be found in the plugin's [package.json](https://github.com/cap-js/advanced-event-mesh/blob/main/package.json).
```
i wouldn't like to have to keep the line number up to date

Reply:
Yes, also thought about that, was also a bit hesitant :)